Gordon360/Models/ViewModels/EmergencyCotnactViewModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
```

Comment:
The name of this file is misspelled.  Please correct it in a simple follow-up pull request.